minos/common/configuration/config.py

```diff
@@ -28,11 +35,17 @@
 COMMANDS = collections.namedtuple("Commands", "broker database items queue")
 REST = collections.namedtuple("Rest", "broker endpoints")
 
+REPOSITORY = collections.namedtuple("Repository", "database user password host port")
+
 
 class MinosConfigAbstract(abc.ABC):
     __slots__ = "_services", "_path"
 
-    def __init__(self, path: str):
+    _default: t.Optional[MinosConfigAbstract] = None
```

Comment:
I agree with you, this is a bad combination with `__slots__`. To avoid problems, I performed the following change: https://github.com/Clariteia/minos_microservice_common/pull/111/commits/f867feeff163f25db96a6db8c83c8e5dd0acade2